src/fragment-generation-utils.js

```diff
@@ -1581,14 +1688,23 @@ const expandRangeEndToWordBound = (range) => {
 /**
  * Helper to determine if a node is a block element or not.
  * @param {Node} node - the node to evaluate
- * @return {Boolean} true iff the node is an element classified as block-level
```

Comment:
Just FYI (because you'll probably see it other places in the code) "iff" is not a typo but a shortened form of "if and only if."

(It's fine to change this, though; I don't think it really adds anything to specify iff here)